### ISSUE TITLE
js standards

### ISSUE THREAD

**Issue by iffyuva** [code-mancers/styleguide#1]
should we just use this https://github.com/feross/standard and call it a day?

**Comment by kgrz** [comment]
Yep. I saw that today and I felt it's well thought out. That or AirBnB. Although AirBnB mandates semicolons and that can be a PITA.

**Comment by emilsoman** [comment]
This looks very basic and fails to answer questions like "how should <insert language feature here> look like" . I like Airbnb's guide because it's more comprehensive.

**Comment by iffyuva** [comment]
+1 for AirBnB convention also. i love semicolons :)
